src/js/components/ConfirmationModal.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const ModalConfirm = (props) => {
+  return (
+    <div className={'confirmation-blurred-blocker'+ (props.showing ? ' showing-blurred-blocker' : '')}>
+      <div className="confirmation-modal">
+      <div>
+        <h4>Are you sure?</h4>
```

Comment:
I think it would be nice to give them some indication of the current progress, especially if has been signed ... eg "Your current submission has been signed. Are you sure you want to refile?"